rudder-web/src/main/resources/rudder-apache-common.conf

```diff
@@ -4,6 +4,11 @@
   Header add X-Chrome-Exponential-Throttling "disable"
 </LocationMatch>
 
+# Force using latest mode in IE.
+<LocationMatch "^/(?!rudder/api)">
```

Comment:
The other one is specific to Rudder "autoreload" on error feature. We could exclude the API (it is done in 4.1), but there is not reason to apply it on the manual or ncf-builder.